### ISSUE TITLE
When Cocoapods is not installed, xcode-project shows "Failed to set code signing settings for x" with no further information, even with verbose output

### ISSUE THREAD

**Issue by roblframpton** [codemagic-ci-cd/cli-tools#276]
I am trying to build a Flutter project.  I have successfully added my profiles to my keychain,
but when I run `xcode-project use-profiles` I get the error:

```
Failed to set code signing settings for _[my_project]_.xcodeproj
```

Sadly the `--verbose` option doesn't give any extra output.  Does anyone know how I can debug this issue?

MacOS 12.0.1
XCode 13.4.1
xcode-project version 0.34.1
Python 3.8

------------------

**EDIT: I think I resolved this.**  The problem was that I was using a fresh machine and the `xcodeproj` gem was not installed, which in turn was because I had yet to install CocoaPods.  It would be nice at the _very least_ if the verbose option actually gave me the ruby script output, since this helped me diagnose it.  Even better would be a message to tell me that `xcodeproj` or Cocoapods needs installed.  I'm leaving this ticket open and renaming it to reflect this.

**Comment by priitlatt** [comment]
@roblframpton thanks for pointing this out. It definitely shouldn't be a burden on the end user to figure out what is causing such problems. We'll see what can be done regarding this and keep you posted here.

**Comment by priitlatt** [comment]
Fix for this was released with version [0.34.3](https://github.com/codemagic-ci-cd/cli-tools/releases/tag/v0.34.3).
